### PR TITLE
Solving Issue #3055

### PR DESCRIPTION
**Pull request by j0rt3g4**
Adding link as @juntinha suggested

### CODE REVIEW

File: windows/security/threat-protection/windows-defender-antivirus/configure-extension-file-exclusions-windows-defender-antivirus.md [hunk 1]
```diff
@@ -46,9 +50,7 @@ This means the exclusion lists have the following characteristics:
 
 >[!IMPORTANT]
 >The use of wildcards such as the asterisk (\*) will alter how the exclusion rules are interpreted. See the [Use wildcards in the file name and folder path or extension exclusion lists](#use-wildcards-in-the-file-name-and-folder-path-or-extension-exclusion-lists) section for important information about how wildcards work.
->
 >You cannot exclude mapped network drives. You must specify the actual network path.
->
```
Comment (illfated) [inline windows/security/threat-protection/windows-defender-antivirus/configure-extension-file-exclusions-windows-defender-antivirus.md]: Judging by the looks of the **&#9432; important** sections in https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-defender-antivirus/configure-extension-file-exclusions-windows-defender-antivirus, I don't think it is practical to remove the blank lines from these sections. The blank lines are there for readability. (see also the next **&#9432; important** section)

File: windows/security/threat-protection/windows-defender-antivirus/configure-extension-file-exclusions-windows-defender-antivirus.md [hunk 2]
```diff
@@ -20,6 +20,10 @@ ms.date: 12/10/2018
 
 - [Windows Defender Advanced Threat Protection (Windows Defender ATP)](https://go.microsoft.com/fwlink/p/?linkid=2069559)
 
+>[!IMPORTANT]
+> [Windows Defender ATP](https://docs.microsoft.com/windows/security/threat-protection/windows-defender-atp/defender-compatibility-windows-defender-advanced-threat-protection) does not adhere to the Windows Defender Antivirus Exclusions settings. In other words, Windows Defender Exclusions,however they are created, are not used by ATP.
+
+
```
Comment (illfated) [inline windows/security/threat-protection/windows-defender-antivirus/configure-extension-file-exclusions-windows-defender-antivirus.md]: 1 blank line after the quoted **&#9432; important** section is enough. The existing **&#9432; important** sections use only 1 blank line after.

File: windows/security/threat-protection/windows-defender-antivirus/configure-extension-file-exclusions-windows-defender-antivirus.md [hunk 3]
```diff
@@ -57,13 +59,13 @@ The exclusions apply to [scheduled scans](scheduled-catch-up-scans-windows-defen
...
 >[!IMPORTANT]
 >Exclusion list changes made with Group Policy **will show** in the lists in the [Windows Security app](windows-defender-security-center-antivirus.md#exclusions).
->
 >Changes made in the Windows Security app **will not show** in the Group Policy lists.
 
 By default, local changes made to the lists (by users with administrator privileges, including changes made with PowerShell and WMI) will be merged with the lists as defined (and deployed) by Group Policy, Configuration Manager, or Intune. The Group Policy lists will take precedence in case of conflicts.
 
 You can [configure how locally and globally defined exclusions lists are merged](configure-local-policy-overrides-windows-defender-antivirus.md#merge-lists) to allow local changes to override managed deployment settings.
 
+
```
Comment (illfated) [inline windows/security/threat-protection/windows-defender-antivirus/configure-extension-file-exclusions-windows-defender-antivirus.md]: This blank line is redundant. I don't think it adds anything useful with regards to formatting (it also extends the length of the affected source needlessly).

**Comment by illfated** [comment]
BTW, cosmetic detail: your @ link to Justinha is missing a letter (should be easy to just type single letters at a time after the @ sign to find the matching name).

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/security/threat-protection/windows-defender-antivirus/configure-extension-file-exclusions-windows-defender-antivirus.md [hunk 4]
```diff
@@ -20,6 +20,10 @@ ms.date: 12/10/2018
 
 - [Windows Defender Advanced Threat Protection (Windows Defender ATP)](https://go.microsoft.com/fwlink/p/?linkid=2069559)
 
+>[!IMPORTANT]
```
Comment (nenonix) [inline windows/security/threat-protection/windows-defender-antivirus/configure-extension-file-exclusions-windows-defender-antivirus.md]:
```suggestion
> [!IMPORTANT]
```

**Comment by mypil** [comment]
@j0rt3g4 - please merge the commit on the copy/edit review made by illfated / JohanFreelancer9 / nenonix  or make the necessary changes on the PR following the suggestions made. Thank you.

File: windows/security/threat-protection/windows-defender-antivirus/configure-extension-file-exclusions-windows-defender-antivirus.md [hunk 5]
```diff
@@ -20,6 +20,10 @@ ms.date: 12/10/2018
...
+> [Windows Defender ATP](https://docs.microsoft.com/windows/security/threat-protection/windows-defender-atp/defender-compatibility-windows-defender-advanced-threat-protection) does not adhere to the Windows Defender Antivirus Exclusions settings. In other words, Windows Defender Exclusions,however they are created, are not used by ATP.
```
Comment (nenonix) [inline windows/security/threat-protection/windows-defender-antivirus/configure-extension-file-exclusions-windows-defender-antivirus.md]:
```suggestion
> [Windows Defender Advanced Threat Protection ](https://docs.microsoft.com/windows/security/threat-protection/windows-defender-atp/defender-compatibility-windows-defender-advanced-threat-protection) does not adhere to Windows Defender Antivirus exclusion settings. This means that any Windows Defender exclusions, no matter how you created them, are not applied by Windows Defender ATP.
```

File: windows/security/threat-protection/windows-defender-antivirus/configure-extension-file-exclusions-windows-defender-antivirus.md (hunk 2 above)
Comment (nenonix) [inline windows/security/threat-protection/windows-defender-antivirus/configure-extension-file-exclusions-windows-defender-antivirus.md]:
```suggestion
```

File: windows/security/threat-protection/windows-defender-antivirus/configure-extension-file-exclusions-windows-defender-antivirus.md (hunk 3 above)
Comment (nenonix) [inline windows/security/threat-protection/windows-defender-antivirus/configure-extension-file-exclusions-windows-defender-antivirus.md]:
```suggestion
```

**Comment by j0rt3g4** [comment]
Thank you @illfated  & @nenonix
@mypil  All changes have been applied.

**Comment by illfated** [comment]
You still need to add back the removed blank lines outside of the added note.
I will add a comment on each, trying to add the blank line back for easier updating.

File: windows/security/threat-protection/windows-defender-antivirus/configure-extension-file-exclusions-windows-defender-antivirus.md [hunk 6]
```diff
@@ -46,9 +49,7 @@ This means the exclusion lists have the following characteristics:
 
 >[!IMPORTANT]
 >The use of wildcards such as the asterisk (\*) will alter how the exclusion rules are interpreted. See the [Use wildcards in the file name and folder path or extension exclusion lists](#use-wildcards-in-the-file-name-and-folder-path-or-extension-exclusion-lists) section for important information about how wildcards work.
->
```
Comment (illfated) [inline windows/security/threat-protection/windows-defender-antivirus/configure-extension-file-exclusions-windows-defender-antivirus.md]: Unfortunately, "Applying suggestions on deleted lines is not supported."
Please add back this missing line.

File: windows/security/threat-protection/windows-defender-antivirus/configure-extension-file-exclusions-windows-defender-antivirus.md [hunk 7]
```diff
@@ -46,9 +49,7 @@ This means the exclusion lists have the following characteristics:
...
 >You cannot exclude mapped network drives. You must specify the actual network path.
->
```
Comment (illfated) [inline windows/security/threat-protection/windows-defender-antivirus/configure-extension-file-exclusions-windows-defender-antivirus.md]: Please add back this missing line.

File: windows/security/threat-protection/windows-defender-antivirus/configure-extension-file-exclusions-windows-defender-antivirus.md [hunk 8]
```diff
@@ -57,7 +58,6 @@ The exclusions apply to [scheduled scans](scheduled-catch-up-scans-windows-defen
 
 >[!IMPORTANT]
 >Exclusion list changes made with Group Policy **will show** in the lists in the [Windows Security app](windows-defender-security-center-antivirus.md#exclusions).
->
```
Comment (illfated) [inline windows/security/threat-protection/windows-defender-antivirus/configure-extension-file-exclusions-windows-defender-antivirus.md]: Please add back this missing line.

**Comment by j0rt3g4** [comment]
@illfated  Not sure if those were the lines... please check.

**Comment by illfated** [comment]
Well, that was not the correct lines. I will push a PR to your own branch in a few moments to replace the right ones.

**Comment by j0rt3g4** [comment]
@illfated  Thank you (that was so cool!) I didn't even know that could be done

**Comment by illfated** [comment]
Sure enough, it is a very nice feature to use when it gets too complicated to explain in words.
(I need to create another PR to you, because I just noticed I misplaced 1 of my changes.)

**Comment by j0rt3g4** [comment]
@illfated yeah, we are all humans...
Let me know where is it to fix it...

**Comment by illfated** [comment]
Locate the following text (somewhere between line 47 and 55):
```
>[!IMPORTANT]
>
>The use of wildcards such as the asterisk (\*) will alter how the exclusion rules are interpreted. See the [Use wildcards in the file name and folder path or extension exclusion lists](#use-wildcards-in-the-file-name-and-folder-path-or-extension-exclusion-lists) section for important information about how wildcards work.
>
>You cannot exclude mapped network drives. You must specify the actual network path.
```
The blank line with the `>` character needs to be moved down from under `>[!IMPORTANT]` and to the line after `>You cannot exclude mapped network drives [ ... ]`

**Comment by j0rt3g4** [comment]
like this?
image

**Comment by illfated** [comment]
Not quite, the inserted line + the `>` character needs to be between
```
>You cannot exclude [...]
```
and
```
>Folders that are reparse points
```

**Comment by j0rt3g4** [comment]
image

I think is good now.. right?

**Comment by illfated** [comment]
Yes, that looks correct to me now.

**Comment by j0rt3g4** [comment]
Ok commited :) thank you!

**Comment by nenonix** [comment]
@illfated apologies, will not do that again!

**Comment by j0rt3g4** [comment]
@nenonix  Relax :) we are all humans but it seems weird to me to remove it but you're the Doc experts

**Comment by illfated** [comment]
@nenonix - Oh. I didn't even notice that you were involved. No problem, at least not that I could see.

**Comment by j0rt3g4** [comment]
@illfated Yeah! I'm not the one that removes lines; I use to leave them for readability

**Comment by nenonix** [comment]
@illfated We really appreciate all your help, I personally have learned so much from your advice.

**Comment by illfated** [comment]
You are very welcome. I am just happy to be able to help, whether it is by posting suggestions or offering advice. 👍

**Comment by mypil** [comment]
@Justinha - Can you please assist to check, approve and merge the content for this PR based on issue #3055?

Thank you.

**Pull request merged**